qlty-config/src/config/builder.rs

```diff
@@ -160,11 +159,40 @@ impl Builder {
             all_exclude_patterns.extend(config.ignore_patterns.clone());
         }
 
+        if !config.ignore.is_empty() {
+            eprintln!(
+                "{} The `{}` field in qlty.toml is deprecated. Please use `{}` or `{}` instead.",
+                style("WARNING:").bold().yellow(),
+                style("ignore").bold(),
+                style("exclude").bold(),
+                style("exclude_patterns").bold()
+            );
+
+            for ignore in &config.ignore {
+                if ignore.file_patterns.is_empty() {
+                    eprintln!(
+                        "{} The use of `{}` field in qlty.toml without {} is no longer supported. Skipping ignore without file_pattern.",
+                        style("WARNING:").bold().yellow(),
+                        style("ignore").bold(),
+                        style("file_patterns").bold()
+                    );
+                    continue;
```

Comment:
@marschattha can you please remind me what an `[[ignore]]` without `file_patterns` can be used for today in the current CLI? Can they do anything useful?

Reply:
But we should have this functionality back one we finish up triage work.

Reply:
Ah, so these would become `[[triage]]` blocks in the new system, right?